tracks/crystal/exercises/pangram/mentoring.md

```diff
@@ -0,0 +1,56 @@
+### Algorithm
+
+There are lots of ways to solve this one. Methods that operate on the whole collection are preferred to iterative methods.
+
+```crystal
+module Pangram
+  ALPHABET = ('a'..'z').to_a
+  def self.pangram?(phrase)
+    (ALPHABET - phrase.downcase.chars).empty?
+  end
+end
+```
+```crystal
+module Pangram
+  ALPHABET = ('a'..'z').to_a
+  def self.pangram?(phrase)
+    ALPHABET & phrase.downcase.chars == ALPHABET
+  end
+end
+```
+```crystal
+module Pangram
+  ALPHABET = Set.new('a'..'z')
+  def self.pangram?(phrase)
+    ALPHABET.subset? Set.new(phrase.downcase.chars)
+  end
+end
+```
+Iterative:
+```crystal
+class Pangram
+  ALPHABET = ('a'..'z')
+
+  getter chars
+
+  def self.pangram?(phrase)
+    self.new(phrase).valid?
+  end
+
+  def initialize(phrase)
+    @chars = phrase.downcase.chars
+  end
+
+  def valid?
+    ALPHABET.all?(&->chars.includes?(Char))
+  end
+end
+```
+
+### Structure
```

Comment:
Please use the following as per the [README.md](https://github.com/exercism/website-copy/blob/master/README.md) of this repository:
```suggestion
### Reasonable solutions
```

```md
### Common suggestions
```

In this case that might not be optimal for what you wrote; you can add the paragraph to the `Reasonable solutions` section, either at the top or bottom.